tests/sonic_xcvr/test_xcvr_field.py

```diff
@@ -0,0 +1,186 @@
+try:
+    from sonic_platform_base.sonic_xcvr.fields.xcvr_field import (
+        CodeRegField,
+        HexRegField,
+        NumberRegField,
+        RegBitField,
+        RegGroupField,
+        StringRegField,
+    )
+    from sonic_platform_base.sonic_xcvr.codes.xcvr_codes import XcvrCodes
+    from sonic_platform_base.sonic_xcvr.mem_maps.xcvr_mem_map import XcvrMemMap
+except ImportError as e:
+    raise ImportError("%s - required module not found" % e)
```

Comment:
The `try..except` block seems unnecessary. Suggest removing.

Reply:
Done